adaptive/learner/triangulation.py

```diff
@@ -266,7 +266,7 @@ class Triangulation:
         or more simplices in the
     """
 
-    def __init__(self, coords):
+    def __init__(self, coords, *, _check_vertices=True):
```

Comment:
Do we need an underscore if this is already a keyword-only argument?

Reply:
I meant to signal that you should only use this if you know what you're doing; I didn't even add it to the docstring!

Reply:
I think it needs both. It should definitely not be a able to use as a positional keyword by accident nor should it be changed by the user.